.github/tests/spire-oidc-insecure/values.yaml

```
@@ -1,3 +1,21 @@
+spire-server:
+  ingress:
+    enabled: true
+    className: "nginx"
+    annotations:
+      nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+      nginx.ingress.kubernetes.io/ssl-passthrough: "true"
```

```suggestion
      nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
      nginx.ingress.kubernetes.io/ssl-passthrough: "true"
      nginx.ingress.kubernetes.io/ssl-redirect: "true"
      nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
```

If we are going to add annotations here anyhow, I would also add the 2 I suggest here. That will improve user experience if people don't specify `https` explicitly they will be auto-redirected.